libs/prov/prov_k8s_cortx_deploy.py

```diff
@@ -1293,53 +1293,53 @@ def test_deployment(self, sns_data, sns_parity,
                                                               common_const.RGW_CONTAINER_NAME,
                                                               self.deploy_cfg["rgw_rpm"])
                             assert_utils.assert_true(resp[0], resp[1])
-            if self.deployment_type not in self.exclusive_pod_list:
-                if setup_client_config_flag:
-                    LOGGER.info("Setting the current namespace")
-                    resp_ns = master_node_list[0].execute_cmd(
-                        cmd=common_cmd.KUBECTL_SET_CONTEXT.format(namespace),
-                        read_lines=True)
-                    LOGGER.debug("response is %s,", resp_ns)
-                    resp = system_utils.execute_cmd(
-                        common_cmd.CMD_GET_IP_IFACE.format(self.deploy_cfg['iface']))
-                    eth1_ip = resp[1].strip("'\\n'b'")
+        if self.deployment_type not in self.exclusive_pod_list:
+            if setup_client_config_flag:
+                LOGGER.info("Setting the current namespace")
+                resp_ns = master_node_list[0].execute_cmd(
+                    cmd=common_cmd.KUBECTL_SET_CONTEXT.format(namespace),
+                    read_lines=True)
+                LOGGER.debug("response is %s,", resp_ns)
+                resp = system_utils.execute_cmd(
+                    common_cmd.CMD_GET_IP_IFACE.format(self.deploy_cfg['iface']))
+                eth1_ip = resp[1].strip("'\\n'b'")
```

Comment:
Is this a specific thing to name it eth1_ip? or can it be more generic?

Reply:
its being specific to pick first interface so just a variable name we thought of using eth1_ip

Reply:
OK. I am fine as far as it's as per the requirement.